src/org/labkey/test/params/FieldDefinition.java

```diff
@@ -302,7 +302,8 @@ public String toString()
     {
         MultiLine("Multi-Line Text", "Multi-Line Text", "string"),
         Integer("Integer", "Integer", "int"),
-        String("Text", "Text (String)", "string"),
+        string("Text", "Text (String)", "string"),
+        String("Text", "Text (String)", "String"), // need upper case String for lookup tableType text
```

Comment:
I would rather not have the duplicate field type definitions here which just differ in text casing. Can we try editing the DomainFieldRow.setLookup so that we change line 383 to `tableType.toUpperCase()` instead?